polkadot/xcm/pallet-xcm/src/lib.rs

```diff
@@ -1449,6 +1499,29 @@ pub mod pallet {
 			})?;
 			Ok(())
 		}
+
+		#[pallet::call_index(13)]
+		#[pallet::weight(T::WeightInfo::execute_blob())]
+		fn execute_blob(
+			origin: OriginFor<T>,
+			message: BoundedVec<u8, MaxXcmEncodedSize>,
```

Comment:
Where is MaxXcmEncodedSize, is this supposed to come from from the pallet config?

Reply:
Yes, pallet config. Right now I had it hardcoded, but leaving it as a config shouldn't be bad